src/components/totals/FundsRaised/__tests__/FundsRaised-test.js

```diff
@@ -255,4 +256,26 @@ describe('FundsRaised', function() {
     });
   });
 
+  describe('Callback function', function() {
```

Comment:
It'd be good to have a slightly more descriptive name for this... callback from what?